site/content/docs/user/known-issues.md

```diff
@@ -322,6 +323,16 @@ systemctl restart firewalld
 See [#1547 (comment)](https://github.com/kubernetes-sigs/kind/issues/1547#issuecomment-623756313)
 and [Docker and Fedora 32 article](https://fedoramagazine.org/docker-and-fedora-32/)
 
+## Fedora 33 SELinux
```

Comment:
Is this one fixed on Fedora 44?
Can we add a note about that?

Last time we added a note it was a mistake to include the fedora version, at the time it didn't apply to older versions but it has applied to newer ones.